src/Nox.Localization/Nox.Localization.csproj

```diff
@@ -30,13 +34,15 @@
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\Nox.Lib\Nox.Lib.csproj" />
+        <ProjectReference Include="..\Nox.Abstractions\Nox.Abstractions.csproj" />
+        <ProjectReference Include="..\Nox.EntityFramework.Sqlite\Nox.EntityFramework.Sqlite.csproj" />
```

Comment:
yup... so lets add all Localization projects to Nox.Localization we are ok in bringing all dependencies, so....